app/models/condition.rb

```diff
@@ -293,4 +297,42 @@ def self.import_from_hash(condition, options={})
 
     return c, status
   end
+
```

Comment:
I had to reinstate this code because the reporting UI still allows expressions containing ruby code to be created.